src/main/java/net/minecraftforge/common/ForgeConfig.java

```diff
@@ -108,10 +108,32 @@
                     .comment("Fix advancement loading to use a proper topological sort. This may have visibility side-effects and can thus be turned off if needed for data-pack compatibility.")
                     .translation("forge.configgui.fixAdvancementLoading")
                     .define("fixAdvancementLoading", true);
+
             builder.pop();
         }
     }
 
+    /**
+     * General configuration that doesn't need to be synchronized but needs to be available before server startup
+     */
+    public static class Common {
+        public final ForgeConfigSpec.ConfigValue<? extends String> defaultWorldType;
+
+        Common(ForgeConfigSpec.Builder builder)
+        {
+            builder.comment("General configuration settings")
+                    .push("general");
+
+            defaultWorldType = builder
+                    .comment("Defines a default world type to use instead of the vanilla 'default' default.")
```

Comment:
Might I suggest a rewording of this, to avoid the doubled-up "default" word?
```suggestion
                    .comment("Defines a default world type to use. The vanilla default world type is represented by 'default'.")
```